### ISSUE TITLE
DAP "run | debug" code lens is being showed by default

### ISSUE THREAD

**Issue by atk91** [scalameta/nvim-metals#498]
### Task

With _almost_ [default setup](https://github.com/scalameta/nvim-metals/discussions/39) the `run | debug` is being shown at the end of each `object Main extends App` line.

It renders just the same font as code, so it does confuse user a little bit, and you cannot simply copy-paste code from terminal since it will include this `run | debug`.

As @ckipp01 said in Discord:
> I need to dig in further and see if at its core this is Neovim sending a request right in the beginning or not

**Comment by atk91** [comment]
If I have `nvim-dap` not installed - `run | debug` is not being displayed

If I have it installed - it is being displayed regardless of whether `require("metals").setup_dap()` is being called or not.

**Comment by ckipp01** [comment]
> If I have it installed - it is being displayed regardless of whether require("metals").setup_dap() is being called or not.

This is _sort_ of expected. The tricky this is that Metals needs to know at initialize time if the client supports debugging or not, so to cheat this I check to see if `nvim-dap` is installed and if it is, I then send in that debugging is supported (even though it may not be set up). If I'd wait to detect the `setup_dap` call it'd delay everything and make the start process quite a bit trickier. That's not a good trade off imo.

**Comment by atk91** [comment]
I agree, however I think it would be nice to add something like "If you do not want to see those `run | debug` do not install `nvim-dap`" in the documentation.

or maybe add a configuration option which will override `nvim-dap`'s presence and make `nvim-metals` act like debugging is not supported?
